tests/unit/argparser/test_colors.py

```diff
@@ -0,0 +1,48 @@
+from unittest import TestCase
```

Comment:
We should create a github action to run the tests for every pr, do you think is a good idea?

Reply:
Automated tests are always a good idea

Reply:
Added the issue #27